commons/src/main/kotlin/com/simplemobiletools/commons/helpers/Constants.kt

```diff
@@ -185,6 +185,13 @@ const val SHOW_PRIVATE_CONTACTS = "show_private_contacts"
 const val MERGE_DUPLICATE_CONTACTS = "merge_duplicate_contacts"
 const val FAVORITES_CONTACTS_ORDER = "favorites_contacts_order"
 const val FAVORITES_CUSTOM_ORDER_SELECTED = "favorites_custom_order_selected"
+const val VIEW_TYPE = "view_type"
+const val CONTACTS_GRID_COLUMN_COUNT = "favourites_column_count"
+
+// contact grid view constants
+const val CONTACTS_GRID_DEFAULT_COLUMNS_COUNT = 3
```

Comment:
default grid count should depend on the screen size, make it more flexible. Check how is it done in the file manager or other apps.